dowhy/gcm/util/plotting.py

```diff
@@ -166,3 +167,51 @@ def _plot_causal_graph_networkx(
 
 def _calc_arrow_width(strength: float, max_strength: float):
     return 0.2 + 4.0 * float(abs(strength)) / float(max_strength)
+
+
+def bar_plot(
+    values: Dict[str, float],
+    uncertainties: Optional[Dict[str, Tuple[float, float]]] = None,
+    ylabel: str = "",
```

Comment:
It would be nice to have the xlabel as well. But not must have.

Reply:
I'll add them in a follow-up PR.